cristin-biobank/src/main/java/no/unit/nva/biobank/common/DomainUriUtils.java

```diff
@@ -0,0 +1,59 @@
+package no.unit.nva.biobank.common;
+
+import static nva.commons.core.paths.UriWrapper.HTTPS;
+import java.net.URI;
+import java.net.URLEncoder;
+import java.nio.charset.StandardCharsets;
+import nva.commons.core.StringUtils;
+import nva.commons.core.paths.UriWrapper;
+
+public final class DomainUriUtils {
+
+    private static final String BIOBANK_PATH_ELEMENT = "biobank";
+
+
+    private DomainUriUtils() {
+    }
+
+    public static URI getBiobankUri(String domainName, String basePath) {
+        return getBaseUriWrapper(domainName, basePath).addChild(BIOBANK_PATH_ELEMENT).getUri();
+    }
+
+    /**
+     * Building up URI ID for Biobank.
+     * @param domainName - env
+     * @param basePath - path
+     * @param identifier - code
+     */
+    public static URI getBiobankUri(String domainName, String basePath, String identifier) {
+        var urlEncodedIdentifier = URLEncoder.encode(identifier, StandardCharsets.UTF_8);
+        var uriString = getBaseUriWrapper(domainName, basePath)
+                   .addChild(BIOBANK_PATH_ELEMENT)
+                   .toString();
+        return URI.create(uriString + "/" + urlEncodedIdentifier);
+    }
+
+    /**
+     * Generating URIs for fields of the Biobank, considering their specific path.
+     * @param domainName - env
+     * @param basePath - path
+     * @param identifier - code
+     * @param pathElement -path element for particular parameter
+     */
+    public static URI getBiobankParamUri(String domainName, String basePath, String identifier, String pathElement) {
+        var urlEncodedIdentifier = URLEncoder.encode(identifier, StandardCharsets.UTF_8);
```

Comment:
Url encoding the cristinBiobankId is not necessary as it can only contain digits.